### DISCUSSION TITLE
[1999770]莱莎的炼金工坊3

### DISCUSSION THREAD

**Discussion by ctlsllk** [pjy612/SteamManifestCache#233] [Share]
appid: 1999770
```
1999771;87ed3357e30334409bab57c0c6fa71f01c33910fdcc448fdb380f2ea7ffa16e9
```

[1999770莱莎的炼金工坊3.txt](https://github.com/pjy612/SteamManifestCache/files/13809477/1999770.3.txt)
第一次使用这个网站，如果不对马上改
内测码C3F703E890ED8FED
联系方式2929826347@qq.com

**Comment by pjy612** [comment]
请把密钥列出来，参考版内顶置

**Reply by ctlsllk** [reply]
谢谢大佬谢谢大佬
